README.md

```diff
@@ -10,3 +10,17 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## Proof
+
+We can prove this through contridiction
+
+Let $A$ and $B$ both be graphs, and $A$ = $(V_1, E_1)$, $B$ = $(V_2, E_2)$
+Say that $A$ contains 5 nodes, while $B$ contains 6.
```

Comment:
When you assume this, it can't be a general proof anymore. Do you need this assumption?